build.gradle

```diff
@@ -188,6 +191,11 @@ processResources {
     from("gatk-launch")
 }
 
+processTestResources {
+    exclude "**/large/**"
+    exclude "**/org/broadinstitute/hellbender/**"
```

Comment:
it was wasting time putting all of our data files into a test jar, we probably shouldn't have stuck all of our test data in test resources... but the symmetry is nice

added a comment